modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring global access for an Ingress Controller on GCP
+An Ingress Controller created on GCP with an internal load balancer generates an internal IP address for the service. A cluster administrator can specify the global access option, which enables clients in any region within the same VPC network and compute region as the load balancer, to reach the workloads running on your cluster.
+
+For more information, please reference the GCP documentation for link:https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access[Global access].
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on GCP infrastructure.
+* You configured an ingress controller to use an internal load balancer.
+* You installed the `oc` command line interface.
++
+[NOTE]
+====
+You can also create an ingress controller and specify the global access option.
+====
+
+.Procedure
+
+. Configure the Ingress Controller resource to allow `Global` access.
++
+.. Configure the Ingress Controller resource:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator edit ingresscontroller/default
+----
++
+.. Edit the YAML file:
++
+.Sample `clientAccess` configuration to `Global`
+[source,yaml]
+----
+  spec:
+    endpointPublishingStrategy:
+      loadBalancer:
+        providerParameters:
+          gcp:
+            clientAccess: Global <1>
+          type: GCP
+        scope: Internal
+      type: LoadBalancerService
+----
+<1> Add `Global` for the `clientAccess` value.
++
+. Run the following command to see that the service allows global access:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress operator edit svc/router-default -o yaml
```

Comment:
Not critical (since this is a technical comment) , but usually it would be better to use a command that outputs values when you're verifying something, as opposed to editing again just to view.